cmds/internet/main.go

```diff
@@ -215,6 +219,19 @@ func configureZOS() error {
 			}
 		}
 
+		if env.NTPServer != nil && *env.NTPServer != "" {
+			// override /etc/ntp.conf
+			servers := strings.Split(*env.NTPServer, ",")
+			var buffer strings.Builder
+			for _, server := range servers {
```

Comment:
No validation on the server variable before generating a broken ntp config file?

Reply:
hmm other than not empty i think we can also validate if the server is suffixed with `.pool.ntp.org`?